test/cmr/opendap/tests/system/app/ous/core.clj

```diff
@@ -34,6 +34,26 @@
     (is (= ["http://e4ftl01.cr.usgs.gov:40510/dir-replaced-by-tags/ASTT/AST_L1T.003/2001.11.29/AST_L1T_00311292001175440_20150303161825_63101.hdf.nc"]
            (util/parse-response response)))))
 
+(deftest no-tag-replacement
+  (let [collection-id "C1200241219-DEMO_PROV"
```

Comment:
Question: who owns this collection (and granule)? How likely is it that it could be deleted or changed and cause this test to fail?

If that's a concern, you can ingest the same data using the HMR_TME provider ... that provider is protected.

Reply:
DEMO_PROV is owned by the CMR team. I think it's fine to use here for the short term until we're able to change the tests over to not rely on a CMR environment.